wgsl/index.bs

```diff
@@ -6137,6 +6137,13 @@ See [[#builtin-inputs-outputs]] for how to declare a built-in variable.
       <td width="50%">The current invocation's [=workgroup ID=],
           i.e. the position of the workgroup in the [=workgroup grid=].
 
+  <tr><td>`num_workgroups`
+      <td>compute
+      <td>in
+      <td>vec3&lt;u32&gt;
+      <td width="50%">The number of workgroups
```

Comment:
Please define this as equal to vec3<u32>( group_count_x, group_count_y, group_count_z ) as those values are defined at the definition of [=compute shader grid=].